.github/workflows/build.yml

```
@@ -1,14 +1,17 @@
 name: build
 
 on:
+  pull_request:
   release:
     types: [released]
   workflow_dispatch:
 
 jobs:
-  publish:
+  build:
     uses: OpenAstronomy/github-actions-workflows/.github/workflows/publish_pure_python.yml@v1
     with:
       upload_to_pypi: ${{ (github.event_name == 'release') && (github.event.action == 'released') }}
+      test_extras: tests
+      test_command: python -c "from asdf import open, AsdfFile, info, config_context, get_config, ValidationError, IntegerType, Stream, ExternalArrayReference"; pytest --pyargs asdf
```

```suggestion
      test_command: pytest --pyargs asdf
```
One minor suggestion. I vote for dropping these import checks from here. If they're useful to include lets add them as a test (which will run with the `pytest --pyargs asdf`) but that can go in a separate PR.